.editorconfig

```diff
@@ -0,0 +1,13 @@
+# EditorConfig info: https://editorconfig.org/
+root = true
+
+[Makefile*]
+indent_style = tab
+
+[*.{c,cpp,h}]
```

Comment:
Our current C++ binding files use a .cxx or .hxx suffix. So we should probably add those.

Note that we are probably going to move the C++ bindings to another repo. But there may be C++ code in the host implementations.